wsmaster/che-core-api-factory-bitbucket-server/src/main/java/org/eclipse/che/api/factory/server/bitbucket/HttpBitbucketServerApiClient.java

```diff
@@ -309,7 +311,7 @@ private BitbucketUser getUser(Optional<String> token)
       throw new ScmCommunicationException(e.getMessage(), e);
     }
 
-    String username = getUsername(response);
+    String username = URLDecoder.decode(getUsername(response), UTF_8);
```

Comment:
can we cover this with tests? my understanding that it was a culprit

Reply:
done